src/main/java/spypunk/tetris/ui/view/GameEntry.java

```diff
@@ -0,0 +1,35 @@
+package spypunk.tetris.ui.view;
+
+public class GameEntry{
```

Comment:
GameEntry iyi bir isim değil bu class için.

Belki `HighScoreEntry` daha niteleyici bir isimlendirme olabilir.

Reply:
dediğiniz gibi yaptım